src/test/resources/org/broadinstitute/hellbender/utils/hmm/hmm-unit-test-generate-truth.R

```diff
@@ -0,0 +1,22 @@
+library(HMM)
```

Comment:
If you do keep these output files, add a comment about how to generate them to the HMM class incase something is updated in the future.